pulp_smash/utils.py

```diff
@@ -52,6 +53,20 @@ def get_broker(server_config):
     )
 
 
+def http_get(url, **kwargs):
```

Comment:
I could have named this function as `get` but I'm predicting we may need `ssh_get` function and `http_get` makes the code more meaningful when reading the test code.